hugegraph-api/src/main/java/com/baidu/hugegraph/api/graph/VertexAPI.java

```diff
@@ -462,7 +462,7 @@ public void checkUpdate() {
             }
             if (this.id != null) {
                 newProps[appendIndex++] = T.id;
-                newProps[appendIndex++] = this.id;
+                newProps[appendIndex] = this.id;
```

Comment:
prefer to keep `appendIndex++` for avoid new code trap

Reply:
OK, add a comment for it, now it the plus is unused